app/controllers/heroku/resources_controller.rb

```diff
@@ -10,6 +10,11 @@ def create
         oauth_grant_code: oauth_grant_code,
       )
 
+      ExchangeGrantTokenJob.perform_later(
```

Comment:
Is there not an `enqueue` method? Or am I think of Resque/Sidekiq?

Reply:
We are using ActiveJob here so that it's easy to swap out the backend job processor no matter what. This is an ActiveJob method: http://edgeguides.rubyonrails.org/active_job_basics.html#enqueue-the-job